baseplate/clients/redis.py

```diff
@@ -92,9 +94,35 @@ class RedisContextFactory(ContextFactory):
 
     """
 
-    def __init__(self, connection_pool: redis.ConnectionPool):
+    PROM_PREFIX = "bp_redis_pool"
+    PROM_LABELS = ["pool"]
+
+    totalConnections = Gauge(
+        f"{PROM_PREFIX}_connections",
```

Comment:
Thoughts on making this (and clustered redis) `f"{PROM_PREFIX}_size"` for consistency with memcached and sqlalchemy implementations?

Reply:
I wonder if we shouldn't do it the other way around, since it describes the contents of the pool.

The redis client in baseplate.go seems to have `connections_total`, `connections_idle`. Having something end in `_total` signals that it's a Counter instead of a Gauge, so I'd keep that out - but switching the other two.